Client/Cliqz/Foundation/Extensions/AppDelegateExtension.swift

```diff
@@ -64,10 +63,27 @@ extension AppDelegate {
             
         }
     }
+
     // Cliqz: Change the status bar style and color
-    static func changeStatusBarStyle(_ statusBarStyle: UIStatusBarStyle, backgroundColor: UIColor) {
-        UIApplication.shared.setStatusBarStyle(statusBarStyle, animated: false)
-		UIApplication.shared.delegate?.window??.backgroundColor = backgroundColor
+	func changeStatusBarStyle(_ statusBarStyle: UIStatusBarStyle, backgroundColor: UIColor, _ isNormalMode: Bool) {
```

Comment:
Could you add a input name for `isNormalMode` to make the code more readable when calling this function